src/main/java/teammates/ui/template/CourseEditSectionRow.java

```diff
@@ -135,19 +137,42 @@ public ElementTag getToggleSessionLevelInSectionButton() {
         
         return permissionInputGroup;
     }
-
+    
+    /**
+     * Creates a row of selections elements (checkboxes), each representing a single section
+     *   and automatically checks a single checkbox if special privileges have been assigned to
+     *   the section it corresponds to.
+     *   
+     * @param sectionNames the list of sections in the course
+     * @param sectionIndex the index of the section which will be checked for special privileges
+     * @return             a list of checkboxes, separated into rows and columns
+     */
     private List<List<ElementTag>> createSpecialSectionsForSectionRow(List<String> sectionNames, 
```

Comment:
Done locally, not pushed yet.

Should the permission input groups in the same class be renamed as well? They serve a similar purpose. An image showing the methods (under the original names) and their related UI components is as follows:

![sectionrow](https://cloud.githubusercontent.com/assets/7546744/15203206/e69eb564-1833-11e6-831c-c6bc08573351.png)